boundary_layer_default_plugin/config/operators/gcp_pubsub_pull_sensor.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Etsy Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+# see: https://airflow.readthedocs.io/en/stable/_modules/airflow/contrib/sensors/pubsub_sensor.html
+
+name: gcp_pubsub_pull_sensor
+operator_class: PubSubPullSensor
+operator_class_module: airflow.providers.google.cloud.sensors.pubsub
```

Comment:
Could you update to `airflow.contrib.sensors.pubsub_sensor` that's the path for 1.10.x.  The new path will be part of Airflow 2.0

Reply:
Done